server.js

```diff
@@ -22,7 +22,7 @@ app.set('view engine', 'ejs')
 const client = new stytch.Client({
         project_id: process.env.STYTCH_PROJECT_ID,
         secret: process.env.STYTCH_SECRET,
-        env: stytch.envs.test,
+        env: stytch.envs.live,
```

Comment:
Why the change to `live`? I think the other projects I've looked at have been `test`.

Reply:
Oops, unintentional, fixed!